src/components/all.scss

```diff
@@ -480,6 +480,31 @@ img {
   border: 2px solid var(--accent);
 }
 
+.custom-details {
+  background-color: var(--accent);
+  border-color: var(--accent);
+  color: var(--text-primary);
+  border-radius: 4px;
+  border-width: 1px;
+  border-style: solid;
+  text-decoration: none;
+  cursor: pointer;
+
+  font-weight: 600;
+  font-size: 16px;
+  flex-shrink: 0;
+  line-height: 1.25;
+  text-decoration: none;
+  white-space: nowrap;
+  padding: 0.5rem 0.75rem;
+  margin: 1rem 0;
+
+  summary {
+    font-size: 1.25rem;
+    list-style-image: url(https://s3-us-west-2.amazonaws.com/s.cdpn.io/3/right-arrow.svg)
```

Comment:
this is a code smell x 50

Reply:
it it's in SVG you are able to inline it

Reply:
Forgot to remove this, thanks